src/executeCommandLine/executeCommandLine.ts

```diff
@@ -104,10 +107,20 @@ namespace ts {
             return `\x1b[1m${str}\x1b[22m`;
         }
         function blue(str: string) {
-            return `\x1b[34m${str}\x1b[39m`;
+            if (supportsRicherColors) {

```

Comment:
Since these strings aren't human-readable, can you leave a comment explaining what the difference will be?